Capfile

```diff
@@ -1,2 +1,29 @@
-%w(bundler deploy rbenv passenger setup).each { |r| require "capistrano/#{r}" }
+# frozen_string_literal: true
+# Load DSL and set up stages
+require 'capistrano/setup'
+
+# Include default deployment tasks
+require 'capistrano/deploy'
+
+# Include tasks from other gems included in your Gemfile
+#
+# For documentation on these, see for example:
+#
+#   https://github.com/capistrano/rvm
+#   https://github.com/capistrano/rbenv
+#   https://github.com/capistrano/chruby
+#   https://github.com/capistrano/bundler
+#   https://github.com/capistrano/rails
+#   https://github.com/capistrano/passenger
+#
+require 'capistrano/rvm'
+# require 'capistrano/rbenv'
+# require 'capistrano/chruby'
+require 'capistrano/bundler'
+require 'capistrano/rails'
+require 'capistrano/rails/assets'
```

Comment:
`capistrano/rails` already requires `capistrano/rails/assets` and `capistrano/rails/migrations`